pkg/controller/secretgenerator/secretgenerator_controller.go

```diff
@@ -20,6 +25,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	resourceRetryInterval = 10 * time.Second
+	resourceRetryTimeout  = 1 * time.Minute
+
+	secretLabel = "secret-generator.opendatahub.io/secretName"
```

Comment:
The [convention for kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels) labels is using [kebab case](https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats) (secret-name) instead of camel case (secretName). Btw I would use the `SECRET_NAME_ANNOTATION` constant instead of defining a new one.

Reply:
`SECRET_NAME_ANNOTATION` doesn't define the secret resource [name](https://github.com/samuelvl/secretgenerator-testing/blob/main/secret-generator/base/secret.yaml#L7) . Also this label will no longer be required if the `OAuthClient` name is kept the same as secret name, so I will remove it.